docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -112,13 +112,79 @@ Comparisons will be with the GBD results of incidence, prevalence, and mortality
 4.0 Intervention
 ++++++++++++++++
 
-
-
 .. _4.1:
 
 4.1 Simulation scenarios
 ------------------------
 
+.. _4.1.1:
+
+4.1.1 Outreach
+~~~~~~~~~~~~~~
+Under this scenario, the effectiveness of the delivery of SBP-lowering and LDL-c lowering therapies would increase through methods such as a phone call, app, or support clinic. 
+
+**Source information:**
+
+Black patients, when prescribed statin as a new medication within the past 1 year, receive automated phone calls and letters starting 1-2 weeks after prescribing, which encourages them to fill the prescription (primary adherence). Receiving this intervention increases fill and initiation of statin from 26% to 42% of patients. OR for intervention vs control was 2.16 (1.91-2.43). Effectively, patients were twice as likely to initiate medication during the first 30 days if intervention was delivered.  
+[Derose-2013]_
+
+Patients, age 30-60 without IHD, but who met any of the following criteria: current tobacco smoker, LDL-c > 3.37 mmol/L, or SBP >140 mmHg, received access to a non-clinical community health center with nurse-practitioner counseling on diet, tobacco use, and exercise. Telephone follow-up and free YMCA exercise sessions were offered. The comparison group received standard of care. Both groups had medication copays covered. At 1 year, statin adherence had a relative odds of 2.2 (95% CI 1.11-4.2) and blood pressure medication adherence had a relative odds of 2.3 (95% CI 1.39-3.88) compared to the control group. 
+[Becker-2005]_
+
+**Implementation:**
+
+For the first 30 days, new statin and blood pressure prescription adherence is increased according to Derose 2013. Adherence to the medications over 1 year after initiation should be increased according to Becker 2005. Effect can persist beyond the length of study.  
+
+**Scenarios:**
+	- Outreach 1.0: Following a prescription of new statin or blood pressure-lowering medication, the initial medication fill rate is increased by 2x and the medication adherence each month is increased 2x.  
```

Comment:
We need more documentation on the prescription model---who gets prescribed new meds, when and why?  what is the initial fill rate, and how should that be implemented in the simulation?  (Apologies if you have this already and I have missed it.)